3600/3/app/controllers/restaurant_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class RestaurantController < AppController
+  include CommentsHelper
+
+  get '/' do
+    @all_restaurants = Restaurant.all
+    erb :home
+  end
+
+  get '/restaurant/:name' do
```

Comment:
Take a look at REST specification https://restfulapi.net/resource-naming/
According to it as a parameter should be provided `:id` not name